ssc/common_financial.cpp

```diff
@@ -385,15 +367,16 @@ bool dispatch_calculations::compute_outputs( std::vector<double>& ppa)
 	}
 
 	size_t i;
-	double dispatch_factor1 = m_cm->as_double("dispatch_factor1");
-	double dispatch_factor2 = m_cm->as_double("dispatch_factor2");
-	double dispatch_factor3 = m_cm->as_double("dispatch_factor3");
-	double dispatch_factor4 = m_cm->as_double("dispatch_factor4");
-	double dispatch_factor5 = m_cm->as_double("dispatch_factor5");
-	double dispatch_factor6 = m_cm->as_double("dispatch_factor6");
-	double dispatch_factor7 = m_cm->as_double("dispatch_factor7");
-	double dispatch_factor8 = m_cm->as_double("dispatch_factor8");
-	double dispatch_factor9 = m_cm->as_double("dispatch_factor9");
+
+    double dispatch_factor1 = m_dispatch_tod_factors[0];// m_cm->as_double("dispatch_factor1");
```

Comment:
Is the comment intended to indicate the former code? In that case, say `// Formerly m_cm->as_double("dispatch_factor1");`

Reply:
superfluous comments removed